schema/switch.yml

```diff
@@ -242,3 +242,294 @@ properties:
+          type: string
+          format: uc-mac
+        acl-dest-macaddress:
+          description: Specifies the destination MAC address for the filter
+          type: string
+          format: uc-mac
+        acl-dest-macbitmask:
+          description: The mask applied to the destination MAC address
+          type: string
+          format: uc-mac
+        acl-packet-format:
+          description: Identifies the protocol encapsulated in the Ethernet frame by its EtherType
+          type: string
+          enum:
+            - any
+            - untagged-eth2
+            - untagged-802.3
+            - tagged-eth2
+            - tagged-802.3
+        acl-vlanid:
+          description: Specifies a VLAN ID to filter traffic from a specific VLAN
+          type: integer
+          minimum: 1
+          maximum: 4094
+        acl-vid-bitmask:
+          description: The mask applied to the VLAN ID
+          type: integer
+          minimum: 1
+          maximum: 4095
+        acl-ethertype:
+          description: Filters packets based on the custom EtherType field in the Ethernet frame
+          type: string
+          default: "800"
+        acl-ethertype-bitmask:
+          description: The mask applied to the EtherType field
+          type: string
+          default: "FFFF"
+        acl-cos:
+          description: Filters based on the Class of Service (CoS) field in the frame
+          type: integer
+          minimum: 0
+          maximum: 7
+        acl-cos-bitmask:
+          description: The mask applied to the CoS field
+          type: integer
+          minimum: 0
+          maximum: 7
+        acl-ip-type:
+          description: Filters traffic based on the IP protocol type (none, IPv4, or IPv6)
+          type: string
+          enum:
+            - none
+            - ipv4
+            - ipv6
+        acl-ipv4-source-address:
+          description: The IPv4 address of the source to filter on
+          type: string
+          format: ipv4
+        acl-ipv4-source-subnetmask:
+          description: The subnet mask applied to the source IPv4 address
+          type: string
+          format: ipv4
+        acl-ipv4-dest-address:
+          description: The IPv4 address of the destination to filter on
+          type: string
+          format: ipv4
+        acl-ipv4-dest-subnetmask:
+          description: The subnet mask applied to the destination IPv4 address
+          type: string
+          format: ipv4
+        acl-ip-proto:
+          description: Filters based on the IP protocol number
+          type: integer
+          minimum: 0
+          maximum: 255
+        acl-ip-source-port:
+          description: Specifies the source port number for filtering
+          type: integer
+          minimum: 0
+          maximum: 65535
+        acl-ip-source-port-bitmask:
+          description: The mask applied to the source port number
+          type: integer
+          minimum: 0
```

Comment:
The comment is once again about the semantic stuff: we excluded 4095 - reserved vlan - but allowing here for port to start from 0;
This right here should either stay 0, but VID should be capped at 4095, or we can have this value start from 1, and leave max VID to be 4094;

Reply:
I agree @Cahb, the purpose of Port 0 is different, and should not be used here specially when we instruct a IP port number to be on an ACL. Thanks for catching this.